### PR TITLE
Add documentation about burndown and computed field

### PR DESCRIPTION
**Pull request by magarnier**
Add documentation on burndown and computed field

### CODE REVIEW

File: languages/en/user-guide/tracker.rst [hunk 1]
```diff
@@ -886,6 +886,67 @@ e-mail notification is sent in response to the import.
     your original regional settings. Import the "fixed" .csv file into
     Tuleap.
 
+Complex field
+-------------
+
+Computed field
+``````````````
+In Tuleap, computed fields are special fields who allows you to to the sum of
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: that allows you

File: languages/en/user-guide/tracker.rst [hunk 2]
```diff
@@ -886,6 +886,67 @@ e-mail notification is sent in response to the import.
...
+all your child field in a tracker hierarchy.
+
+A common use case for this field is calculation of remaining effort in release.
+Release remaining_effort is the sum of sprint remaining_effort
+and sprint remaining effort is the sum of user stories
+
+Computation rules:
+ - we never compute twice the same node,
+   if a user story is linked to two sprint, its remaining effort will be add once in calculation
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: Why ? I understand that point if it's linked two time to the same story.

File: languages/en/user-guide/tracker.rst [hunk 3]
```diff
@@ -886,6 +886,67 @@ e-mail notification is sent in response to the import.
...
+ - manual value break the calculation,
+   when calculation encounter a manual value we never computed children of node, we keep manual value for computation.
+ - permissions are not taken in account during calculation process,
+   user will see the remaining effort global, even he can't see some artifacts.
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: even if he/she

File: languages/en/user-guide/tracker.rst (hunk 2 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Previously we had a bug:
if a story was linked in two different sprint, the remaining effort was counted twice in the release calculation.

File: languages/en/user-guide/tracker.rst [hunk 4]
```diff
@@ -886,6 +886,67 @@ e-mail notification is sent in response to the import.
...
+
+
+Burndown field
+``````````````
+Burndown is a graphical representation of remaining effort,
+and is used to track team progress.
+In burndown every dot represent a remaining effort for a given day.
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: represents

File: languages/en/user-guide/tracker.rst [hunk 5]
```diff
@@ -886,6 +886,67 @@ e-mail notification is sent in response to the import.
...
+
+Burndown display is based on a cache table:
+ - every night yesterday value is computed and cached
+ - the value for the day "today" is never cached and calculated at every display
+ - if burndown has missing day, we generate the full burndown
+ - if start_date or duration is updated, the full burndown is calculated again
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: start date

File: languages/en/user-guide/tracker.rst [hunk 6]
```diff
@@ -886,6 +886,67 @@ e-mail notification is sent in response to the import.
...
+Complex field
+-------------
+
+Computed field
+``````````````
+In Tuleap, computed fields are special fields who allows you to to the sum of
+all your child field in a tracker hierarchy.
+
+A common use case for this field is calculation of remaining effort in release.
+Release remaining_effort is the sum of sprint remaining_effort
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: remaining effort

File: languages/en/user-guide/tracker.rst [hunk 7]
```diff
@@ -886,6 +886,67 @@ e-mail notification is sent in response to the import.
...
+
+With a server located in Paris
+
+========================== =====================
+Server time                31th July 2017 8:00AM
+Today remaining_effort     10
+========================== =====================
+
+My team B, update the remaining_effort to 9.
+
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: There is something missing between your last sentence and the table

File: languages/en/user-guide/tracker.rst [hunk 8]
```diff
@@ -886,6 +886,67 @@ e-mail notification is sent in response to the import.
...
+========================== ========================
+
+With a server located in Paris
+
+========================== =====================
+Server time                31th July 2017 8:00AM
+Today remaining_effort     10
+========================== =====================
+
+My team B, update the remaining_effort to 9.
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: Team B updates the ...

File: languages/en/user-guide/tracker.rst [hunk 9]
```diff
@@ -886,6 +886,67 @@ e-mail notification is sent in response to the import.
...
+
+========================== ===============================
+in Montreal                value for 31th July will be 9.
+in Tokyo                   value for 31th July will be 10.
+in Tokyo                   value for 1th August will be 9.
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: I don't understand this value for Tokyo here

File: languages/en/user-guide/tracker.rst (hunk 1 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst (hunk 2 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst (hunk 3 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst (hunk 4 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst (hunk 6 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst (hunk 8 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst (hunk 9 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst (hunk 7 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst [hunk 10]
```diff
@@ -886,6 +886,81 @@ e-mail notification is sent in response to the import.
     your original regional settings. Import the "fixed" .csv file into
     Tuleap.
 
+
+Complex field
+-------------
+
+Computed field
+``````````````
+In Tuleap, computed fields are special fields that allows you to to the sum of
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: to do the sum

File: languages/en/user-guide/tracker.rst [hunk 11]
```diff
@@ -886,6 +886,81 @@ e-mail notification is sent in response to the import.
...
+all your child field in a tracker hierarchy.
+
+A common use case for this field is calculation of remaining effort in release.
+Release remaining effort is the sum of sprint remaining effort
+and sprint remaining effort is the sum of user stories
+
+Computation rules:
+ - we never compute twice the same node,
+   if a user story is linked to two sprint, the release remaining effort add only one time the user story remaining effort
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: two sprints

File: languages/en/user-guide/tracker.rst [hunk 12]
```diff
@@ -886,6 +886,81 @@ e-mail notification is sent in response to the import.
...
+ - permissions are not taken in account during calculation process,
+   user will see the remaining effort global, even he/she can't see some artifacts.
+
+Example of computation work:
+
+.. figure:: ../images/screenshots/computed.png
+   :align: center
+   :alt: Example of computed field calculation
+
+- User story "6 remaining effort is count just once for release, but sprint #2 and #3 reflects correct remaining effort
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: User story #6

File: languages/en/user-guide/tracker.rst (hunk 10 above)
Comment (magarnier) [inline languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst [hunk 13]
```diff
@@ -886,6 +886,81 @@ e-mail notification is sent in response to the import.
...
+ - permissions are not taken in account during calculation process,
+   user will see the remaining effort global, even he/she can't see some artifacts.
+
+Example of computation work:
+
+.. figure:: ../images/screenshots/computed.png
+   :align: center
+   :alt: Example of computed field calculation
+
+- User story #6 remaining effort is count just once for release, but sprint #2 and #3 reflects correct remaining effort
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: is count => is used
sprint #2 and #3 reflects => reflect without s

File: languages/en/user-guide/tracker.rst [hunk 14]
```diff
@@ -886,6 +886,81 @@ e-mail notification is sent in response to the import.
...
+- If I manually set remaining effort for Sprint #2 to 10, release remaining effort will be 60 (10+30+20)
+- I am connected a a member who only can see release, I will see 60 as remaining effort
```
Comment (yannis-rossetto) [inline languages/en/user-guide/tracker.rst]: a a member ?

File: languages/en/user-guide/tracker.rst (hunk 13 above)
Comment (magarnier) [reply languages/en/user-guide/tracker.rst]: Done

File: languages/en/user-guide/tracker.rst (hunk 14 above)
Comment (magarnier) [reply languages/en/user-guide/tracker.rst]: Done

**Pull request merged**